crypto/rsa/rsa_lib.c

```diff
@@ -13,7 +13,6 @@
  */
 #include "internal/deprecated.h"
 
-#include <stdio.h>
```

Comment:
Since you're cleaning up this file, can I ask you to change `#include "openssl/param_build.h"` to `#include <openssl/param_build.h>` and place it in proper order?

Reply:
sure, fixup pushed